content/en/docs/components/multi-tenancy/getting-started.md

```diff
@@ -113,14 +113,14 @@ Kubeflow {{% kf-latest-version %}} provides automatic profile creation:
   - The Kubeflow deployment process automatically creates a profile for the user
```

Comment:
I recommend changing this bullet point to: 
- Automatic profile creation is not activated by default, and needs to be explicitly included as part of deployment. Turning on automatic user profile creation will automatically create a profile for the user performing the deployment. You will then be able to see this user profile in the dropdown list of the Kubeflow central dashboard.

Reply:
Optionally you can add: 
If you do not activate automatic user profile creation, you will need to manually create profiles for users or projects. For more information, see [[Manual Profile Creation](docs/components/multi-tenancy/getting-started/#manual-profile-creation).